src/nginxconfig/templates/setup_sections/ssl_cert.vue

```diff
@@ -159,13 +183,54 @@ THE SOFTWARE.
                         ].filter(x => x !== null).join(' ')
                     )).join('\n');
             },
+            acmeShCmds() {
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            'acme.sh --issue',
+                            `-d ${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            `-w ${domain.server.path.computed}${domain.server.documentRoot.computed}`,
+                        ].filter(x => x !== null).join(' ')
+                    )).join('\n');
+            },
+            acmeCopy() {
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            `cp -R /root/.acme.sh/${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            this.$props.data.global.https.letsEncryptCertRoot.computed.replace(/\/+$/, '') + '/' + domain.server.domain.computed,
```

Comment:
Maybe I'm incorrect about this. but this step appears to be trying to expose the certs in the correct location for nginx. [According to the acme.sh docs, you should not be copying directly from `~/.acme.sh/`](https://github.com/acmesh-official/acme.sh#3-install-the-cert-to-apachenginx-etc).

Based on the docs, I would guess this needs to be something like:

```suggestion
                            `acme.sh --install-cert`,
                            `-d ${domain.server.domain.computed}`,
                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
                            '--keyfile ' + letEncryptPath + domain.server.domain.computed + '/privkey.pem',
                            '--fullchain-file ' + letEncryptPath + domain.server.domain.computed + '/fullchain.pem',
                            '--reloadcmd \'nginx -t && systemctl reload nginx\'',
```

Reply:
I am a bit confused about this myself. The cert can just be added via this:

Reply:
The docs explicitly state that the cert should **not** be directly added from `~/.acme.sh/`, you should use the command to copy them.

Reply:
I reread the acme.sh myself. I misunderstood it for a second thinking it mean to not point to it’s file location for a second 😅

I will make the change and the rest here! Thanks y’all for the feedback so far and I will commit some changes in a bit!